cartography/data/jobs/cleanup/gcp_labels_cleanup.json

```diff
@@ -0,0 +1,17 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (n:GCPLabel)<-[:LABELED]-(m)<-[*]-(:GCPProject{id: {GCP_PROJECT_ID}})<-[:OWNER]-(:CloudanixWorkspace{id: {WORKSPACE_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
I like the approach you have taken here. Instead of cleaning up labels one resource at a time, doing all of them at once.